games-rpg/baldurs-gate-ee/Manifest

```diff
@@ -1 +1 @@
-DIST gog_baldur_s_gate_enhanced_edition_2.5.0.9.sh 2877124067 BLAKE2B 244e93ae089703c9fae3dcfcad4fb9c1eb32695e7288f7a05b3f2996887f90d6912e745e9b47a93ba45ee3ddda8363e69d22e62c57258ee54c24c47f07b2f619 SHA512 56d4796a959658f29cabef6e7339db5ba9fa792d9fd899ebc375c9b5cf94305b6d11c21d41a8809fcb1cbe38ecb4bc7152158ef19c87395374ab9060f48756a6
```

Comment:
1. Please mention the new version in commit message.
2. Please use split commits for adding new version and removing old.
3. Please don't remove old version immediately unless there's a good reason to.